src/contracts/core/DelegationManager.sol

```diff
@@ -231,9 +238,9 @@ contract DelegationManager is
             "DelegationManager.undelegate: caller cannot undelegate staker"
```

Comment:
require allocation delay to register

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/695/commits/287cdceb9484cf4257bfcec2da74319bba547fe1